barretenberg/cpp/src/barretenberg/vm/avm/generated/circuit_builder.hpp

```diff
@@ -10,31 +10,38 @@ namespace bb {
 
 class AvmCircuitBuilder {
   public:
+    // Do not use this constant directly, use get_circuit_subgroup_size() instead.
+    constexpr static size_t CIRCUIT_SUBGROUP_SIZE = 1 << 21;
```

Comment:
Any reason to make this public if you don't want it used?

Reply:
Ha, good catch. Yes, because I wanted to do a `static_assert` in execution.cpp :)